HyperVExtension/src/HyperVExtensionServer/Program.cs

```diff
@@ -97,6 +98,8 @@ private static void HandleCOMServerActivation()
         // This could be called by either of the COM registrations, we will do them all to avoid deadlock and bind all on the extension's lifetime.
         using var extensionServer = new Microsoft.Windows.DevHome.SDK.ExtensionServer();
         var hyperVExtension = Host.GetService<HyperVExtension>();
+        var hyperVManager = Host.GetService<IHyperVManager>();
+        hyperVManager.ImportHyperVModule();
```

Comment:
Can we do this from HyperVManager itself (from constructor or lazy initialization whenever needed)? Doesn't look like clients of IHyperVManager need to know about that detail.

Reply:
removed and provided a better work around. I'll need to investigate what happened between .Net 6 and .Net 8 in dev home that caused this. I created a test app with .Net 8 and PowerShell SDK 7.4.0 and it worked fine. So there is something weird with what we're doing in Dev Home. But I'll investigate it. For now I found that attempting to load it via Get-Module when the session starts and then using it again to get the list of available modules to find the module we want fixes the issue. This happened last minute unfortunately as we were using .Net 6 and PowerShell sdk 7.2.* before then